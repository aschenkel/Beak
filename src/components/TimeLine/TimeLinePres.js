import React, { Component } from 'react';
import {
  View,
  Text,
  Image,
  Dimensions,
  StyleSheet,
  Alert,
  AsyncStorage,
  ScrollView
} from 'react-native';
import Loader from '../Loaders/Loader'


class TimeLinePres extends Component{

    render(){
        return (
               <Text>{this.props.avgGrade}}</Text>
            )
    }

}

export default TimeLinePres