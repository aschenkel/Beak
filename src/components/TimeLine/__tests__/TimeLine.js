import React from 'react';
import TimeLinePres from '../TimeLinePres';
import renderer from 'react-test-renderer';

it('renders user timeline', () => {
  const tree = renderer.create(
    <TimeLinePres tweets={{text:"first"},{text:"second tweet"},{text:"3rd"}}
            grade={[2,5,1]}
        />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
