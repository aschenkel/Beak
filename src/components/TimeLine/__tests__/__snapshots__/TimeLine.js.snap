// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders user timeline 1`] = `
<View>
  <RCTScrollView
    data={
      Object {
        "text": "3rd",
      }
    }
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    onViewableItemsChanged={undefined}
    renderItem={[Function]}
    renderScrollComponent={[Function]}
    scrollEnabled={false}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</View>
`;
